risc0/groth16/src/prove/mod.rs

```diff
@@ -25,6 +25,8 @@ use anyhow::Result;
 
 use crate::Seal;
 
```

Comment:
lets put a comment on this re-export to ourselves in the future to make sure we don't remove this line.